src/test-run/commands/assertion-executor-factory.js

```diff
@@ -0,0 +1,133 @@
+import { assert, expect } from 'chai';
```

Comment:
Move it from `commands` directory. I guess it's worth to move executor to `src/assertions/index.js`